terminus-ui/pipes/src/truncate/truncate.pipe.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { TsTruncateAtPipe } from './truncate.pipe';
+
+describe(`TsTruncateAtPipe`, function() {
```

Comment:
Once we're using the ellipsis character, we should add a couple expects that verify the length of the string too